configuration/messageproperties/messages_en.properties

```diff
@@ -1048,6 +1052,7 @@ pihcore.genitalUlcer=Genital ulcer
 pihcore.withoutCondom=Without condom
 pihcore.analSex=Anal sex
 pihcore.homoSexual=Male homosexuality activity
+pihcore.homosexual=Men who have sex with other men
```

Comment:
@jmbabazi I would recommend that you change the code for pihcore.homosexual.  It's confusing to have another one with different text but uppercase S "homoSexual".  Maybe change to menSexWithMen.  Thanks.

Reply:
Agree with @lnball